mono/metadata/object.h

```diff
@@ -36,7 +36,7 @@ typedef void*    (*MonoCompileFunc)	     (MonoMethod *method);
 typedef void	    (*MonoMainThreadFunc)    (void* user_data);
 
 #define MONO_OBJECT_SETREF(obj,fieldname,value) do {	\
-		g_assert (sizeof((obj)->fieldname) == sizeof (gpointer*));	\
+		assert (sizeof((obj)->fieldname) == sizeof (gpointer*));	\
```

Comment:
When I made this same change in our local branch, I had to include <assert.h> at the top of this file.